doc/content/getting-started/aws/ecs/updating/_index.md

```diff
@@ -0,0 +1,56 @@
+---
+title: "Updating"
+description: ""
+weight: 3
+---
+
+This page describes the steps for updating {{% tts %}} on AWS ECS.
+
+<!--more-->
+
+Most steps involve updating a CloudFormation stack from the templates. Some steps involve some manual work.
+
+## Overview
+
+From AWS' point of view, updating {{% tts %}} can be divided into three parts:
+
+1. CloudFormation templates
+
+2. Docker images
+
+3. Migrations
+
+## Updating CloudFormation stacks
+
+Updating CloudFormation stacks can be done via the AWS console, using the "Create changeset" functionality. All stacks need to be updated in their numbering order. For example, a stack using template `1-1-vpc` needs to be updated before `1-2-bastion`, which needs to be updated before `2-1-db-aurora-master`.
+
+Not all stacks require a replacement in each version update. Please refer to `UPGRADING.md`, which contains per-version changes in templates. Only templates mentioned in that file have changed in given version. Moreover use that file as a general guideline what changes to expect.
+
+Make sure deployed templates correspond to your {{% tts %}} version. When updating, pay extra attention to:
+- new parameters
+- removed parameters
+- parameters with changed possible values
+
+as default parameters might not be valid in context of your deployment.
+
+### `2-5-db-timescale`
```

Comment:
Can you please add a short sentence as a simple introduction explaining why you are talking about these specific templates?